todo-app/src/app/todo-list/todo-list.component.html

```diff
@@ -0,0 +1,99 @@
+<div class="wrapper">
+  <form class="form" [formGroup]="form" (submit)=" $event.preventDefault()" (keydown.enter)="$event.preventDefault()">
```

Comment:
Would be better to move this form toa  new component

Reply:
having trouble

Reply:
I believe you could deal with all issues :)